app/views/shared/check_answers/_item.html.erb

```diff
@@ -1,6 +1,9 @@
-<% read_only = local_assigns[:read_only] ? read_only : false %>
+<%
+read_only = local_assigns[:read_only] ? read_only : false
+no_border = local_assigns[:no_border] ? '--no-border' : ''
+%>
 
-<div class="govuk-summary-list__row normal-word-break" id="app-check-your-answers__<%= name %>">
+<div class="govuk-summary-list__row<%= no_border %> normal-word-break govuk-!-margin-bottom-2" id="app-check-your-answers__<%= name %>">
```

Comment:
This will add the `govuk-!-margin-bottom-2` class to all check answers items. Regardless of the `no_border` parameter.
Are you sure this is what we want?

Reply:
It shouldn't be there thanks.